src/Storages/Freeze.cpp

```diff
@@ -26,11 +24,16 @@ void FreezeMetaData::save(DiskPtr data_disk, const String & path) const
 
     writeIntText(version, buffer);
     buffer.write("\n", 1);
-    writeBoolText(is_replicated, buffer);
-    buffer.write("\n", 1);
-    writeBoolText(is_remote, buffer);
-    buffer.write("\n", 1);
-    writeString(replica_name, buffer);
+    if (version == 1) {
+        /// is_replicated and is_remote are not used
+        bool is_replicated = true;
+        writeBoolText(is_replicated, buffer);
+        buffer.write("\n", 1);
+        bool is_remote = true;
+        writeBoolText(is_remote, buffer);
+        buffer.write("\n", 1);
```

Comment:
I would prefer to completely remove these flags without changing version. It will break compatibility with `22.6.1`, but I think it's not a problem, because `22.6` was just released recently and I doubt that someone has already started to use `22.6.1` with this freeze-metadata-stuff in production. And previous serialization format was invalid anyway (I mean issues with escaping).

Reply:
The thing is, flags were introduced before this PR. See history of removed code for details